AndroidSDK/src/com/leanplum/Leanplum.java

```diff
@@ -573,7 +573,7 @@ public void noPendingDownloads() {
       });
 
       // Reduce latency by running the rest of the start call in a background thread.
-      Util.executeAsyncTask(new AsyncTask<Void, Void, Void>() {
+      Util.executeAsyncTask(true, new AsyncTask<Void, Void, Void>() {
```

Comment:
what does true mean here? can u add javadoc to Util.executeAsyncTask please? the varargs make it hard to understand.....